src/github/issue-comment.ts

```diff
@@ -83,7 +83,10 @@ export const issueCommentWebhookHandler = async (
 
 const syncIssueCommentsToJira = async (jiraHost: string, context: WebhookContext, gitHubInstallationClient: GitHubInstallationClient) => {
 	const { comment, repository, issue } = context.payload;
-	const { body: gitHubMessage, id: gitHubId, html_url: gitHubCommentUrl } = comment;
+	const { id: gitHubId } = comment;
+	const gitHubLogin: string = comment.user?.login || "undefined";
+	const gitHubMessage: string = comment.body || "undefined";
```

Comment:
probably better to use ?? , otherwise an empty string will also be mapped to undefined